### PR TITLE
upstream merge

### PR DESCRIPTION
**Pull request by Carbonhell**
don't merge,it's my first attempt at this, didn't even test it yet

### CODE REVIEW

**Comment by ghost** [comment]
Conflicts, good job

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,66 @@
 # This list auto requests reviews from the specified org members
 # when a PR that modifies the file in question is opened
 # This list is alphabetized by User -> Filename KEEP IT THAT WAY
+
+# ChangelingRain
```
Comment (Pyko1) [inline .github/CODEOWNERS]: welp.no

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
-[Directions]: # (If you discovered this issue from playing HippieStation hosted servers
+[Round ID]: # (If you discovered this issue from playing tgstation hosted servers:)
```
Comment (Pyko1) [inline .github/ISSUE_TEMPLATE.md]: uh oh

**Comment by Pyko1** [comment]
the .github/ folder should be our own stuff, prolly

**Comment by Carbonhell** [comment]
conflicts should be fixed,but i still have to:
1) check if anything that shouldn't have been edited got edited
2) test it

**Comment by Manly-Dwarf** [comment]
most of these are actually good changes, but why are we merging monkeymode?

**Comment by ghost** [comment]
You should restart this from scratch and make sure your master is up-to-date before merging upstream

**Comment by Carbonhell** [comment]
@Chnkr if there isn't a way to recover this pr without starting from scratch again... i'll probably drop my attempt,sorry, any clue if there's a way?can't i just git pull from upstream?

**Pull request closed without merging**